### ISSUE TITLE
Recommendations is generating an error

### ISSUE THREAD

**Issue by ryancammer** [docker/scout-action#17]
Hi there! I have the following steps in my action:

```
      - name: Docker Login
        id: docker-login
        uses: docker/login-action@v2
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}

      - name: Docker Scout
        id: docker-scout
        uses: docker/scout-action@v1.0.8
        with:
          command: quickview,cves,recommendations
          image: ${{ env.IMAGE_NAME }}:${{ steps.prep.outputs.BUILD_ID }}
          ignore-unchanged: true
          write-comment: false
          sarif-file: docker-scout-results.output.json

      - name: Upload Docker Scout scan results as artifact
        id: upload-docker-scout-results
        uses: actions/upload-artifact@v2
        with:
          name: docker-scout-results-${{ github.run_id }}.json
          path: docker-scout-results.output.json
```          

And it's generating the following error for recommendations:

```
recommendations
      ✓ SBOM of image already cached, 1718 packages indexed
  panic: runtime error: invalid memory address or nil pointer dereference
  [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x17a7adc]
  
  goroutine 1 [running]:
  github.com/docker/scout-cli-plugin/internal/mk.DiffVulnSummaryVertical(0xc02078a630, 0x0)
  	/home/runner/work/scout-cli-plugin/scout-cli-plugin/internal/mk/mk.go:294 +0x3c
  github.com/docker/scout-cli-plugin/internal/format/recommendations.fillCurrentTag({0xc026e4b9f0, 0xe}, {0xc0123f4718?, 0x8, 0x2?}, 0xc01f325950)
  	/home/runner/work/scout-cli-plugin/scout-cli-plugin/internal/format/recommendations/markdown.go:217 +0xad5
  github.com/docker/scout-cli-plugin/internal/format/recommendations.genMk(0xc01dcfd170, 0xc01dd71720, 0xc00006c800?, {0x20?, 0x11?, {0x0?, 0xc01f325c10?}}, {0xc021513040, 0x3b}, {0x210f51f, ...}, ...)
  	/home/runner/work/scout-cli-plugin/scout-cli-plugin/internal/format/recommendations/markdown.go:81 +0x565
  github.com/docker/scout-cli-plugin/internal/format/recommendations.PrintMarkdown({0x2b021a0, 0xc0207d1120}, 0xc000136080?, 0xc02023c000?, 0x0?, {0x0?, 0x0?, {0x0?, 0x0?}}, {0xc021513040, ...}, ...)
  	/home/runner/work/scout-cli-plugin/scout-cli-plugin/internal/format/recommendations/markdown.go:54 +0x87
  github.com/docker/scout-cli-plugin/internal/gha.(*recommendationsCmd).Run(0x1e5d9a0?, {0x2b15a70, 0xc0005b9e90}, 0xc0008ec000, 0x50b668?, 0xc0003e85b0)
  	/home/runner/work/scout-cli-plugin/scout-cli-plugin/internal/gha/recommendations.go:50 +0x23d
  github.com/docker/scout-cli-plugin/internal/gha.Run({0x2b15a70, 0xc0005b9e90}, 0xc0008ec000, {0xc00004443d, 0xf}, 0xc0005f4170?, 0xc0004e8278?)
  	/home/runner/work/scout-cli-plugin/scout-cli-plugin/internal/gha/commands.go:116 +0x9f
  main.run()
  	/home/runner/work/scout-cli-plugin/scout-cli-plugin/cmd/github-action/main.go:72 +0x4a5
  main.main()
  	/home/runner/work/scout-cli-plugin/scout-cli-plugin/cmd/github-action/main.go:83 +0x13
```  	

I'm happy to provide you with any additional information you need, if I can provide it.

Thank you for your help!

**Comment by eunomie** [comment]
This should be fixed by using [`v1.0.9`](https://github.com/docker/scout-action/releases/tag/v1.0.9)

**Comment by ryancammer** [comment]
Thank you @eunomie! I can verify that this is working now.
